apps/meteor/app/sms/server/SMS.ts

```diff
@@ -31,14 +32,14 @@ export const SMS = {
 	},
 };
 
-settings.watch('SMS_Enabled', function (value) {
+settings.watch<boolean>('SMS_Enabled', function (value) {
 	SMS.enabled = value;
 });
 
-settings.watch('SMS_Default_Omnichannel_Department', function (value) {
+settings.watch<string>('SMS_Default_Omnichannel_Department', function (value) {
 	SMS.department = value;
 });
 
-settings.watch('SMS_Service', (value) => {
+settings.watch<string>('SMS_Service', (value) => {
```

Comment:
Can we please remove these watchers? At this point, it seems unnecessary since these settings are already cached